### PR TITLE
Enhancements and Fix

### PR DESCRIPTION
**Pull request by ispiropoulos**
1. Send renamed to "Select"
2. Dismiss Photo Library Picker when Select is tapped on Image Preview Screen
3. Add a property "isFilePickerEnabled" to enable / disable the File Picker.

### CODE REVIEW

File: ImagePicker/Resource/JVTImageFilePicker.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 @interface JVTImageFilePicker : NSObject <UIDocumentPickerDelegate, UIDocumentMenuDelegate, JVTImagePreviewVCDelegate>
 @property (nonatomic,assign) CGSize imageResizeSize;
+@property (assign) bool isFilePickerEnabled;
```
Comment (mcmatan) [inline ImagePicker/Resource/JVTImageFilePicker.h]: Where is this used?

File: ImagePicker/Resource/JVTImageFilePicker.m [hunk 2]
```diff
@@ -126,17 +130,17 @@ - (void)presentFilesPickerOnController:(UIViewController *)presentFromController
...
     @weakify(self);
-    UIImagePickerController *imagePickerController = [[UIImagePickerController alloc] init];
-    imagePickerController.sourceType = UIImagePickerControllerSourceTypeSavedPhotosAlbum;
-    [self.presentedFromController presentViewController:imagePickerController animated:YES completion:nil];
+    _imagePickerController = [[UIImagePickerController alloc] init];
+    _imagePickerController.sourceType = UIImagePickerControllerSourceTypeSavedPhotosAlbum;
+    [self.presentedFromController presentViewController:_imagePickerController animated:YES completion:nil];
     
-    imagePickerController.finalizationBlock = ^(UIImagePickerController *picker, NSDictionary *info) {
+    _imagePickerController.finalizationBlock = ^(UIImagePickerController *picker, NSDictionary *info) {
```
Comment (mcmatan) [inline ImagePicker/Resource/JVTImageFilePicker.m]: You are using private properties, please change to using getters (self.)

File: ImagePicker/Resource/JVTImageFilePicker.m [hunk 3]
```diff
@@ -147,20 +151,20 @@ - (void)takePhotoOrVideoPress {
     AVAuthorizationStatus authStatus = [AVCaptureDevice authorizationStatusForMediaType:AVMediaTypeVideo];
     if (authStatus == AVAuthorizationStatusAuthorized || authStatus == AVAuthorizationStatusNotDetermined) {
         @weakify(self);
-        UIImagePickerController *imagePickerController = [[UIImagePickerController alloc] init];
```
Comment (mcmatan) [inline ImagePicker/Resource/JVTImageFilePicker.m]: Same here, change from private to "self.imagePickerController"

File: ImagePicker/Resource/JVTImageFilePicker.h (hunk 1 above)
Comment (ispiropoulos) [inline ImagePicker/Resource/JVTImageFilePicker.h]: The vast majority of apps would use this control to just upload a picture, so File upload is not always needed. This property can be set after instantiating the picker in order to hide / show the file picker.

File: ImagePicker/Resource/JVTImageFilePicker.m (hunk 2 above)
Comment (ispiropoulos) [inline ImagePicker/Resource/JVTImageFilePicker.m]: Done.

File: ImagePicker/Resource/JVTImageFilePicker.m (hunk 3 above)
Comment (ispiropoulos) [inline ImagePicker/Resource/JVTImageFilePicker.m]: Done

**Comment by mcmatan** [comment]
I've fixed some more places using _imageFilesPicker as private, and added a default value to "isFilesEnabled" to YES, and merged.
Thank you for your contribution (: have a great day

**Comment by ispiropoulos** [comment]
Thank you for the great control ;)

**Pull request merged**